experimental/framework/tree-react-api/src/packageVersion.ts

```diff
@@ -0,0 +1,9 @@
+/*!
```

Comment:
I don't _think_ this package needs the `packageVersion` stuff. I believe that's something needed for framework packages to determine compatibility at runtime. I don't think this package would need this.

Reply:
Correct. Removed (along with the package script to generate it)